docs/feature_unicode.md

```diff
@@ -1,56 +1,78 @@
 # Unicode Support
 
-There are three Unicode keymap definition methods available in QMK:
+Unicode characters can be input straight from your keyboard! There are some limitations, however.
 
-## `UNICODE_ENABLE`
+There are three different methods available in QMK:
```

Comment:
```suggestion
QMK has three different methods for enabling Unicode input and defining keycodes:
```
Thoughts on this? I think it may be worth stating what these _methods_ are for, since there are also input modes further down the page.

I think it'd be worth having a `## Methods` section (and `### Basic Unicode`, `### Unicode Map`, `### UCIS` within it) along with the existing `## Input Modes` section, since the two classifications are orthogonal.

Reply:
Yeah I think the structure of this page needs a little more focus. but I'll hold off on that for now to avoid scope creep.

Reply:
Sounds good. @ me when you make that PR :)